README.md

```diff
@@ -15,18 +15,19 @@ you should use the [development template](https://github.com/shopware/developmen
 In each commit a composer.lock is contained to ensure that the version being
 deployed is the version that was tested in our CI. We currently provide two
 branches:
-- `6.1`: stable patch releases (`v6.1.0-rc2`, `v6.1.0`, `v6.1.19`, `v6.1.*`, but not `v6.2.0`)
-- `master`: stable minor+patch releases (`v6.1.3`, `v6.1.15`, `v6.2.0`, `v6.3.0`...)
+- `6.1`: stable patch releases (`v6.1.*`)
```

Comment:
could we make the following line more general? otherwise we would have to change it every major release

Reply:
Any idea? :-)

Reply:
An "x" for minor ? 🤔